templates/CRM/Event/Page/EventInfo.tpl

```diff
@@ -75,7 +75,7 @@
 <div class="vevent crm-event-id-{$event.id} crm-block crm-event-info-form-block">
   <div class="event-info">
   {* Display top buttons only if the page is long enough to merit duplicate buttons *}
-  {if array_key_exists('summary', $event) or array_key_exists('description', $event)}
+  {if (array_key_exists('summary', $event) && $event.summary) or array_key_exists('description', $event) && $event.description}
```

Comment:
Technically there should be brackets on the 2nd pair but the truth table happens to work out the same.